internal/tools/aws/database.go

```diff
@@ -176,43 +200,43 @@ func (d *RDSDatabase) rdsDatabaseProvision(installationID string, logger log.Fie
 		return err
 	}
 	if len(vpcs) != 1 {
-		return fmt.Errorf("expected 1 VPC for cluster %s, but got %d", clusterID, len(vpcs))
+		return fmt.Errorf("expected 1 VPC for cluster %s (found %d)", clusterID, len(vpcs))
 	}
 
 	rdsSecret, err := d.client.secretsManagerEnsureRDSSecretCreated(awsID, logger)
 	if err != nil {
 		return err
 	}
 
-	// keyAliasName := KMSAliasNameRDS(awsID)
-	// keyMetadata, err := d.client.kmsGetSymmetricKey(keyAliasName)
-	// if err != nil {
-	// 	if !IsErrorCode(err, kms.ErrCodeNotFoundException) {
-	// 		return errors.Wrapf(err, "unable to verify encryption key %s for the RDS cluster ID %s", keyAliasName, awsID)
-	// 	}
-
-	// 	keyMetadata, err = d.client.kmsCreateSymmetricKey("Key used for encrypting RDS database")
-	// 	if err != nil {
-	// 		return errors.Wrapf(err, "unable to create an encryption key for the RDS cluster ID %s", awsID)
-	// 	}
-
-	// 	err = d.client.kmsCreateAlias(*keyMetadata.KeyId, keyAliasName)
-	// 	if err != nil {
-	// 		scheduleErr := d.client.kmsScheduleKeyDeletion(*keyMetadata.KeyId, KMSMinTimeEncryptionKeyDeletion)
-	// 		if scheduleErr != nil {
-	// 			logger.WithError(scheduleErr).Errorf("Failed to schedule deletion for the encryption key %s", keyAliasName)
-	// 		}
-	// 		return errors.Wrapf(err, "unable to create an encryption key alias name for the RDS cluster ID %s", awsID)
-	// 	}
-	// }
+	kmsResourceNames, err := d.getKMSResourceNames(awsID)
+	if err != nil {
+		return err
+	}
 
-	// if *keyMetadata.KeyState != kms.KeyStateEnabled {
-	// 	return errors.Errorf("expected encryption key %s state to be Enabled, but it was %s", keyAliasName, *keyMetadata.KeyState)
-	// }
+	var keyMetadata *kms.KeyMetadata
+	if len(kmsResourceNames) > 0 {
+		enabledKeys := d.getEnabledEncryptionKeys(kmsResourceNames, logger)
+		switch len(enabledKeys) {
+		case 1:
+			keyMetadata = enabledKeys[0]
+		default:
+			return errors.Errorf("db cluster %s should have exactly one enabled/active encryption key (found %d)", awsID, len(enabledKeys))
+		}
```

Comment:
nit: a bit cleaner this way I think.
```suggestion
		enabledKeys := d.getEnabledEncryptionKeys(kmsResourceNames, logger)
		if  len(enabledKeys) != 1 {
			return errors.Errorf("db cluster %s should have exactly one enabled/active encryption key (found %d)", awsID, len(enabledKeys))
		}
		keyMetadata = enabledKeys[0]
```